commands/azureCommands/delete-image.ts

```diff
@@ -13,38 +13,69 @@ import { AzureImage } from "../../utils/Azure/models/image";
 import { Repository } from "../../utils/Azure/models/repository";
 import * as quickPicks from '../utils/quick-pick-azure';
 
+/** Function to untag an Azure hosted image
+ * @param context : if called through right click on AzureImageNode, the node object will be passed in. See azureRegistryNodes.ts for more info
+ */
+export async function untagAzureImage(context?: AzureImageTagNode): Promise<void> {
+    removeImage(context, true);
```

Comment:
`removeImage` returns Promise<boolean>? Do you need to await and check the return value?